nvidia-network-operator/chart/values.yaml

```diff
@@ -18,6 +18,11 @@
 nebius:
   images:
     ulimitFixer: cr.eu-north1.nebius.cloud/e00vzg8t0148cc4zex/nebius/network-operator-v24-4-0/busybox:1.27.2
+  ulimitFixer:
+    tolerations:
+  #   - key: nvidia.com/gpu
```

Comment:
Should it be commented?

Reply:
I added that example toleration and left it commented as I didn't want to change behavior of the application. Today no tolerations are added to `ulimit-fixer` and wasn't sure if adding it would cause any breaking changes.  

If you think we should add the toleration I can remove the comments.